src/consumption.cpp

```diff
@@ -14,10 +14,13 @@
 #include "mutation.h"
 #include "options.h"
 #include "translations.h"
+#include "units.h"
 
 #include <string>
 #include <algorithm>
 
+using namespace units::literals;
```

Comment:
I'm not clear on what's being protected here, the other machinery in units.h should certainly stay in the units namespace, I'm just suggesting the literal definition move out.
Std puts literally everything in namespaces, which is correct for a library, but we don't need to deal with that in application code.
I'll make a follow-up pr that does the correction if you like (or you can bump it to another pr if you prefer), so this one is good to go I think.  I can't merge right now since I'm travelling.